OTKit/otkit-colors/colors.yml

```diff
@@ -1,3 +1,49 @@
 props:
-  # TODO
-  
\ No newline at end of file
+
+  # Primary Action
+  # This is only used on objects, backgrounds with large text (24px or larger, bold), and will be used as a color to guide users towards actionable content.
+  # =============================================
+  
+  color-primary:
+    value: "#F91F14"
+  
+  color-primary-active:
+    value: "#FB665F"
+  
+  color-primary-disabled:
+    value: "rgba(#F91F14, 0.40)"
```

Comment:
Yes you are right, you need to follow the rgba specs for those values. I see that we also have similar error on one specific value on the [OTTheme color token](https://github.com/opentable/design-tokens/blob/master/OTTheme/ottheme-colors/colors.yml#L64), I'll open an issue for this. Good catch 👍